x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/335_connector_update_configuration.yml

```diff
@@ -72,8 +72,8 @@ setup:
               type: str
               ui_restrictions: [ ]
               validations:
-                - constraint: 0
-                  type: greater_than
+                - constraint: [123, 456, 789]
```

Comment:
It could make sense to list multiple validations here (like you did in the parser unit test), where each one uses a different constraint to have a little bit more coverage.

Reply:
yeah can add it here :+1:

Reply:
Added ✅